egs/reverb/asr1_2ch/run.sh

```diff
@@ -0,0 +1,374 @@
+#!/bin/bash
```

Comment:
It's better to generalize this recipe so that we could use any number of channels by specifying options.